.github/workflows/browser-tests.yml

```diff
@@ -73,11 +78,16 @@ jobs:
 
             - name: Checkout PR branch
               uses: actions/checkout@v4
+              with:
+                  fetch-depth: 2
```

Comment:
What does fetch depth: 2 do? i see above we checkout everything with fetch depth: 0

Reply:
im not of a strong opinion so I'm good with two but I think 1 would work too but again zero opinion. this is non blocking for me.